src/python/WMCore/ReqMgr/DataStructs/DefaultConfig/PERMISSION_BY_STATUS.py

```diff
@@ -0,0 +1,76 @@
+"""
+Data structure that gets uploaded to central CouchDB and is
+then used to control who can access which resources in
+ReqMgr2 and in its CouchDB backend. Note that only the
+PERMISSION_BY_STATUS data structure gets persisted in the database.
+
+A short summary of the permission access is:
+* admin: can make any transition. Usually done by services or WMCore admins.
+* ops: can perform any transition that is not supposed to be made by WMCore services.
+    For the time being, allow staging and staged as well.
+* ppd: can create and approve new requests.
+    For the time being, allow rejected and aborted as well.
+"""
+
+### FIXME TODO: remove these comments
```

Comment:
Note to myself: all these comments need to be removed. I will do so before getting it merged.